WORKSPACE

```diff
@@ -883,7 +883,74 @@ http_archive(
     strip_prefix= "rules_scala-%s" % rules_scala_version
 )
 
-load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repositories")
-scala_repositories()
-load("@io_bazel_rules_scala//specs2:specs2_junit.bzl","specs2_junit_repositories")
-specs2_junit_repositories()
\ No newline at end of file
+SCALA_BUILD_FILE = """
+# scala.BUILD
+java_import(
+    name = "scala-xml",
+    jars = ["lib/scala-xml_2.11-1.0.5.jar"],
```

Comment:
Hi @nwangtw,

Thanks for working on this.

For long-term maintenance perspective, i think the following constants can help us when Scala version is upgraded with just small change (if makes sense)
```
scala_version = "2.11"
scala_point_version = scala_version +".11"
```